src/sentry/quotas/base.py

```diff
@@ -469,3 +469,4 @@ def get_blended_sample_rate(self, project: "Project") -> Optional[float]:
 
         :param project: The project model.
         """
+        return 0.5
```

Comment:
Why do we have the `get_blended_sample_rate` defined here?

Reply:
ooops :)

Reply:
testing :p